Makefile

```diff
@@ -26,11 +27,11 @@ $(gitrepo): tests/setup-git
 	cd tests && ./setup-git
 
 clean:
-	rm -f $(objects) vcprompt
+	$(RM) $(objects) vcprompt
 
 DESTDIR =
 PREFIX = /usr/local
 .PHONY: install
 install: vcprompt
-	install -d $(DESTDIR)$(PREFIX)/bin
-	install -m 755 vcprompt $(DESTDIR)$(PREFIX)/bin
+	install -d $(DESTDIR)/$(PREFIX)/bin
```

Comment:
If DESTDIR does not contain a trailing `/` then the installation process could break.  Even if DESTDIR is empty, having an extra `/` at the beginning of a path does nothing: `//usr` is the same as `/usr`.